vpr/src/pack/cluster.cpp

```diff
@@ -2115,6 +2134,8 @@ static void update_cluster_stats(const t_pack_molecule* molecule,
     molecule_size = get_array_size_of_molecule(molecule);
     cb = nullptr;
 
+    auto& floorplanning_ctx = g_vpr_ctx.mutable_floorplanning();
```

Comment:
will have these passed in as a parameter as a part of the struct I am creating

Reply:
can remove the above to-do comment from Vaughn. Or change the comment to a better overall comment for what the function does.